libraries/tests/AWS.Lambda.Powertools.Metrics.Tests/EMFValidationTests.cs

```diff
@@ -139,20 +139,75 @@ public void When100MetricsAreAdded_FlushAutomatically()
             for (var i = 0; i <= 100; i++)
             {
                 Metrics.AddMetric($"Metric Name {i + 1}", i, MetricUnit.Count);
-            }
 
+                if (i == 100)
```

Comment:
I wanted to make sure and test that the first flush would happen and what are the values